instana/recorder.py

```diff
@@ -87,8 +87,9 @@ def record_span(self, span):
                 service_name = self.agent.options.service_name
                 json_span = SDKSpan(span, source, service_name)
 
-            # logger.debug("Recorded span: %s", json_span)
-            self.agent.collector.span_queue.put(json_span)
+            if json_span.l != 0:
```

Comment:
I'd suggest we do this at the very beginning and ignore span completely if `level` is set to `0`

Reply:
Ooops. Indeed. Fixed.